config/static.go

```diff
@@ -61,7 +61,6 @@ type (
 		SafeBrowsing       SafeBrowsingStaticCfg `yaml:"SafeBrowsing"`
```

Comment:
SafeBrowsing is only used for blacklist urls.  Should we remove code surrounding that as well?  There are references in the config file and the readme too.